tools/fetch_elfutils.sh

```diff
@@ -58,5 +60,5 @@ echo "Compile elfutils using ${C_COMPILER}"
 if [[ "$(basename "${C_COMPILER}")" == clang* ]]; then
   export CFLAGS="-Wno-xor-used-as-pow -Wno-gnu-variable-sized-type-not-at-end -Wno-unused-but-set-parameter"
 fi
-cd ${TARGET_EXTRACT} && ./configure CC=${C_COMPILER} --disable-debuginfod --disable-libdebuginfod --disable-symbol-versioning
-make -j4 -C ${TARGET_EXTRACT}
+./configure CC="${C_COMPILER}" --disable-debuginfod --disable-libdebuginfod --disable-symbol-versioning --prefix "${TARGET_EXTRACT}"
+make "-j$(nproc)" install
```

Comment:
Just thinking out loud here, but `nproc` uses `sched_getaffinity()` to get the process CPU mask.  In a lot of cloud environments, resource isolation is implemented via cycle counting instead of CPU affinity, meaning that `sched_getaffinity()` will overestimate the compute capacity available to the cgroup.

I don't think this matters at all.  In fact, our builds are small enough that `make -j` would probably be fine as well.

Reply:
`make -j` will not limit the number of jobs, I think I still prefer `nproc` even if it is slightly wrong